--- spyder-kernels-2.0.2/spyder_kernels.egg-info/requires.txt~	2021-05-02 17:36:53.000000000 +0000
+++ spyder-kernels-2.0.2/spyder_kernels.egg-info/requires.txt	2021-05-03 13:47:00.499137378 +0000
@@ -1,5 +1,5 @@
 cloudpickle
-decorator<5
+decorator
 jupyter-client>=5.3.4
 pyzmq>=17
 
--- spyder-kernels-2.0.2/setup.py~	2021-04-02 18:07:52.000000000 +0000
+++ spyder-kernels-2.0.2/setup.py	2021-05-03 13:47:25.117137035 +0000
@@ -37,7 +37,7 @@
 
 REQUIREMENTS = [
     'cloudpickle',
-    'decorator<5',  # Higher versions break the Cython magic
+    'decorator',  # Higher versions break the Cython magic
     'ipykernel<5; python_version<"3"',
     'ipykernel>=5.3.0; python_version>="3"',
     'ipython<6; python_version<"3"',
